lib/components/Modal.js

```diff
@@ -67,6 +66,12 @@ var Modal = React.createClass({
     elementClass(document.body).remove('ReactModal__Body--open');
   },
 
+  setNode: function(portalClassName) {
+    this.node = this.node || document.createElement('div');
+    this.node.className = portalClassName;
+    document.body.appendChild(this.node);
```

Comment:
Potentially you would end up appending many children on each `componentWillReceiveProps`, which needs then to be checked before appending.